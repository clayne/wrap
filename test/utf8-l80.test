/dev/null | -l80 | utf8-01.txt
