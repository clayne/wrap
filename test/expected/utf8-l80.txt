This is a text file that contains some Unicode characters — ½ ⅓ ⅔ ¼ ¾ ⅕ ⅖ ⅗ ⅘ ⅛
⅜ ⅝ ⅞ — encoded as UTF-8 for testing purposes.
